docs/reference/FAQ.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: FAQ
+parent: Reference
+nav_order: 4
+permalink: /reference/faq/
+---
+
+- TOC
+{:toc}
+
+# What languages does ClusterFuzz support?
+
+ClusterFuzz definitely supports C, C++ when compiled with
+[clang](https://clang.llvm.org/). It should also support Rust and it probably
+supports other languages that can be compiled with an LLVM-based toolchain (e.g.
+Swift).
+
+## Why is an LLVM-based toolchain required?
+
+An LLVM-based toolchain is required (or at least strongly recommended) for two
+reasons:
+* Because ClusterFuzz relies on LLVM [sanitizers]({{ site.baseurl }}/reference/glossary/#Sanitizer) 
+  for detecting and identifying bugs.
+
+* Because use of [libFuzzer and AFL] in ClusterFuzz requires coverage
+  instrumentation only LLVM can provide.
+
+## What if I really don't want to use an LLVM-based toolchain?
+
+We strongly recommend using an LLVM-based toolchain. That said, you may be able
+to use some parts of ClusterFuzz with [gcc](https://gcc.gnu.org/) or extend
+ClusterFuzz to support your use of gcc. However, if you do this, you are in
+unexplored territory and we can not support this. Gcc can probably be used for
```

Comment:
nit: "Gcc" -> "GCC"

Reply:
I'd also change all "gcc" to "GCC". The official page seems to use all caps version: https://gcc.gnu.org/